esd-group-3/web/addInvoice.jsp

```diff
@@ -0,0 +1,37 @@
+<%-- 
+    Document   : addInvoice
+    Created on : 15-Dec-2020, 18:52:25
+    Author     : Austin
+--%>
+
+<%@page contentType="text/html" pageEncoding="UTF-8"%>
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <title>Add Invoice Page</title>
+    </head>
+    <body>
+<!--  read to read in the patient id, date, starttime, endtime, slot
+        <div>    
+            <select name="Filters" id="filter">
+                <option value="all">All Invoices</option>
+                <option value="nhs">NHS Patient Invoices</option>
+                <option value="private">Private Patient Invoices</option>
+            </select>
+            
+            <label for="start">Start date:</label>
+            <input type="date" id="start" name="start"
+                   value="2018-07-22"
+                   min="2018-01-01" max="2021-12-31">
+            <label for="start">End date:</label>
+            <input type="date" id="end" name="end"
+                   value="2018-07-22"
+                   min="2018-01-01" max="2021-12-31">
+            
+            <form action="InvoiceViewerServlet" method="GET">
+                <input type="submit" value="Update" class="button logout"> 
+            </form>
+        </div>-->
```

Comment:
Need to remove the comments

Reply:
This file isn't finished yet as it's going to be part of another ticket too, so these comments are just here for future me/someone to do.